src/ubu/gii/dass/test/c01/ReusablePoolTest.java

```diff
@@ -27,30 +41,89 @@ public void setUp() throws Exception {
 	 */
 	@After
 	public void tearDown() throws Exception {
+
+		pool = null;
 	}
 
 	/**
 	 * Test method for {@link ubu.gii.dass.c01.ReusablePool#getInstance()}.
+	 * 
+	 * Tests the following conditions:
+	 * - if {@link ubu.gii.dass.c01.ReusablePool#getInstance()} returns not null
+	 * - if two invocations of {@link ubu.gii.dass.c01.ReusablePool#getInstance()}
+	 * 
 	 */
 	@Test
 	public void testGetInstance() {
-		fail("Not yet implemented");
+
+		ReusablePool localPool = ReusablePool.getInstance();
+
+		assertNotNull(pool); // assert pool not null
+		assertNotNull(localPool); // assert localPool not null
+		assertEquals(pool, localPool); // assert getInstance returns the same
+										// ReusablePool instance

```

Comment:
Añadir un mensaje descriptivo que identifique la prueba en la sentencia Assert. En este caso sería "La instancia Singleton no es la misma"

> public static void assertTrue(java.lang.String message,  boolean condition)

    Asserts that a condition is true. If it isn't it throws an AssertionFailedError with the given message.